tests/test-repo.c

```diff
@@ -151,6 +151,50 @@ test_repo_equal (Fixture       *fixture,
   g_assert_false (ostree_repo_equal (closed_repo, closed_repo));
 }
 
+static void
+test_repo_get_min_free_space (Fixture *fixture,
+                              gconstpointer test_data)
+{
+  g_autoptr (GKeyFile) config = NULL;
+  g_autoptr(GError) error = NULL;
+  g_autoptr(OstreeRepo) repo = ostree_repo_create_at (fixture->tmpdir.fd, ".",
+                                                      OSTREE_REPO_MODE_ARCHIVE,
+                                                      NULL,
+                                                      NULL, &error);
+  g_assert_no_error (error);
+
+  const char *values_to_test[] = { "500MB",
+                                   "0MB",
+                                   "17179869185GB", /* Overflow parameter: bytes > G_MAXUINT64 */
+                                   NULL
```

Comment:
It'd be slightly cleaner I think to have this be an array of 2-tuple (struct) `{ const char *val; bool should_succeeed }` and assert that `error` is set if and only if `should_succeed`.

That said, up to you, I don't mind merging as is.

Reply:
Thank you for the suggestion.

Pushed a `fixup` commit.